backend/api/tests.py

```diff
@@ -71,12 +71,13 @@ def test_post_create_view(self):
             response, self.relative_profile_url, status_code=status.HTTP_201_CREATED
         )
 
+class APIRootTestCase(TestCase):
     def test_root_status_code(self):
         """
         Test APIRootView for status code 200.
```

Comment:
@Awakhiwekyle thanks for adding this docstring, it makes the test output look better:

![image](https://github.com/PythonBulawayo/ChitChat/assets/3416132/cf90b566-aa9d-4dd3-a6ce-04ebb9d97e87)